bigquery_etl/shredder/config.py

```diff
@@ -272,6 +272,24 @@ def dataset_id(self):
     glean_target(table="org_mozilla_ios_lockbox_stable.baseline_v1"): IOS_LOCKBOX_SRC,
     glean_target(table="org_mozilla_ios_lockbox_stable.events_v1"): IOS_LOCKBOX_SRC,
     glean_target(table="org_mozilla_ios_lockbox_stable.metrics_v1"): IOS_LOCKBOX_SRC,
+    # activity stream
+    client_id_target(table="activity_stream_stable.events_v1"): DESKTOP_SRC,
+    client_id_target(table="messaging_system_stable.cfr_v1"): DESKTOP_SRC,
+    client_id_target(table="messaging_system_stable.onboarding_v1"): DESKTOP_SRC,
+    client_id_target(table="messaging_system_stable.snippets_v1"): DESKTOP_SRC,
+    client_id_target(table='activity_stream_stable.sessions_v1'): DESKTOP_SRC,
+    client_id_target(table='messaging_system_derived.cfr_users_daily_v1'): DESKTOP_SRC,
+    client_id_target(table='messaging_system_derived.cfr_users_last_seen_v1'): DESKTOP_SRC,
+    client_id_target(table='messaging_system_derived.onboarding_users_daily_v1'): DESKTOP_SRC,
+    client_id_target(table='messaging_system_derived.onboarding_users_last_seen_v1'): DESKTOP_SRC,
+    client_id_target(table='messaging_system_derived.snippets_users_daily_v1'): DESKTOP_SRC,
+    client_id_target(table='messaging_system_derived.snippets_users_last_seen_v1'): DESKTOP_SRC,
+    impression_id_target(table="activity_stream_stable.impression_stats_v1"): IMPRESSION_SRC,
+    impression_id_target(table="activity_stream_stable.spoc_fills_v1"): IMPRESSION_SRC,
+    impression_id_target(table="messaging_system_stable.undesired_events_v1"):IMPRESSION_SRC,
+    impression_id_target(table='messaging_system_derived.cfr_users_daily_v1'): IMPRESSION_SRC,
+    impression_id_target(table='messaging_system_derived.cfr_users_last_seen_v1'): IMPRESSION_SRC,
+    impression_id_target(table='messaging_system_stable.personalization_experiment_v1'): IMPRESSION_SRC,
```

Comment:
@ncloudioj can you confirm that `client_id`/`impression_id` in these tables that map to `client_id`/`payload.scalars.parent.deletion_request_impression_id` in `telemetry_stable.deletion_request_v4`?

Reply:
@relud LGTM except for one special case.

The `message_system_stable.cfr_v1` table has both `client_id` and `impressoin_id` in its schema. We're collecting `impresson_id` in the release channel, and `client_id` in the pre-release channels as well as the CFR experiment audience in the release. They never show up together in a single ping though (except the `deletion-request` ping).

So we also want to add it to the `impression_id_target`.

Reply:
> They never show up together in the same ping though (except the `deletion-request` ping).

Thank you for this bit of information, it was v helpful in getting this working.